playbooks/edx-west/suclass-prod-app.yml

```diff
@@ -31,6 +31,7 @@
       nginx_sites:
         - lms
         - cms
+        - lms-preview
```

Comment:
I wonder if we need a similar entry for `service_variants_enabled` in `edxapp_suclass_vars.yml`...